pkg/oc/cli/secrets/options.go

```diff
@@ -98,11 +99,22 @@ func (o SecretOptions) GetServiceAccount() (*kapi.ServiceAccount, error) {
 func (o SecretOptions) GetSecretNames(secrets []*kapi.Secret) sets.String {
 	names := sets.String{}
 	for _, secret := range secrets {
-		names.Insert(secret.Name)
+		names.Insert(parseResourceName(secret.Name))
 	}
 	return names
 }
 
+// parseResourceName receives either a resource name as either
+// <resource type> / <name> or <name> and returns only the resource <name>.
+func parseResourceName(name string) string {
```

Comment:
This isn't meant to be generic, right? Maybe we could strip the `secret(s)/` prefix explicitly in that case. That would also be more clear in my opinion even if we are certain at this point that `name` has `secret/` as the prefix if it contains a slash. Otherwise, what happens if i `oc secret unlink whatever/steve`?